README.md

```diff
@@ -57,6 +57,14 @@ This example will render "Hello Taylor" into a container on the page.
 
 You'll notice that we used an HTML-like syntax; [we call it JSX](https://react.dev/learn#writing-markup-with-jsx). JSX is not required to use React, but it makes code more readable, and writing it feels like writing HTML.
 
+## Example
+
+```jsx
+
+const element = <h1>Hello, world!</h1>;
+```
+This simple code snippet is neither a string nor HTML. It is called JSX, which is a syntax extension to JavaScript. 
```

Comment:
Examples and definitions are given in the reference link : https://react.dev/learn#writing-markup-with-jsx , so why is it required here ?

Reply:
It's not really an example to use React. The teaching section in react.dev is better suited for this since you can't get an overall idea of React by looking at a single line.